### PR TITLE
ADCS estimator usb console integration

### PR DESCRIPTION
**Pull request by tanishqaggarwal**

### CODE REVIEW

File: python/usb_console/simulation.py [hunk 1]
```diff
@@ -68,6 +72,9 @@ def configure_sim(self):
         self.computer_state_follower, self.computer_state_leader = self.eng.initialize_computer_states('not_detumbled', nargout=2)
         self.main_state_trajectory = []
 
+        self.eng.workspace['const']['dt'] = 120e6  # Control cycle time = 120 ms = 120e6 ns
```
Comment (tanishqaggarwal) [inline python/usb_console/simulation.py:79]: @nhz2 I hope using step = 0.12s doesn't affect the sim results by a lot

Comment (kylekrol) [reply python/usb_console/simulation.py:79]: I don't think anything assumes it was 0.1s and the integrators will probably still be accurate enough.

**Review by kylekrol — APPROVED** [review]
Looks good to me!

**Comment by tanishqaggarwal** [comment]
@kkrol27 I added some new features since your last review.

**Comment by tanishqaggarwal** [comment]
Based on offline understandings, I'm going to treat this as "approved" and merge.

**Pull request merged**